.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
we probably don't want to commit visual studio code files. my suggestion would be to add this to your global `.gitignore`